test/plugin/constants.rb

```diff
@@ -0,0 +1,422 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Constants used by unit tests for Google Cloud Logging plugin
+module Constants
+  include Fluent::GoogleCloudOutput::Constants
+
+  # generic attributes
+  HOSTNAME = Socket.gethostname
+
+  WRITE_LOG_ENTRIES_URI = 'https://logging.googleapis.com/v2beta1/entries:write'
+
+  # attributes used for the GCE metadata service
+  PROJECT_ID = 'test-project-id'
+  ZONE = 'us-central1-b'
+  FULLY_QUALIFIED_ZONE = 'projects/' + PROJECT_ID + '/zones/' + ZONE
+  VM_ID = '9876543210'
+
+  # attributes used for custom (overridden) configs
+  CUSTOM_PROJECT_ID = 'test-custom-project-id'
```

Comment:
[Optional] There are clearly defined sections here, as evidenced by the common constant name prefixes. Eventually, we might want to split this into multiple submodules, each with its own set of constants. But this change is a pure refactoring, so I'd suggest doing that later.